client/state/sharing/keyring/schema.js

```diff
@@ -8,7 +8,7 @@ export const itemSchema = {
 			properties: {
 				ID: { type: 'integer' },
 				additional_external_users: { type: 'array' },
-				expires: { type: 'string' },
+				expires: { type: [ 'number', 'boolean' ] },
```

Comment:
The expiry date is either a Unix timestamp or `false` - this was the closest I could get to represent that.